whipper/common/mbngs.py

```diff
@@ -250,11 +325,19 @@ def _getMetadata(release, discid=None, country=None):
                     track.artist = trackCredit.getName()
                     track.sortName = trackCredit.getSortName()
                     track.mbidArtist = trackCredit.getIds()
+                    track.recordingArtist = recordingCredit.getName()
 
                     track.title = t.get('title', t['recording']['title'])
                     track.mbid = t['id']
                     track.mbidRecording = t['recording']['id']
-                    track.mbidWorks = _getWorks(t['recording'])
+                    works = _getWorks(t['recording'])
+                    track.mbidWorks = _getWorksId(works)
+                    track.composers = _getComposers(works)
+                    track.performers = []
+                    if 'artist-relation-list' in t['recording']:
+                        track.performers = _getPerformers(
+                            t['recording']['artist-relation-list']
+                              )
```

Comment:
This indentation seems off.

Reply:
i just remember having a similar Flake8 error somewhere in the past so i did it this way. I think it was [this one](https://www.flake8rules.com/rules/E124.html)

Reply:
But you’re doing as shown in that link either.

Reply:
:) yeh right oh my brain.